### PR TITLE
Initial version of this role

### PR DESCRIPTION
**Pull request by haslersn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# ansible_cephadm
+
+This role installs [cephadm](https://docs.ceph.com/en/quincy/cephadm/) and Docker on Debian.
```
Comment (SF2311) [inline README.md]:
```suggestion
This role installs [cephadm](https://docs.ceph.com/en/latest/cephadm/) and Docker on Debian.
```
We don't need to pin the version in this link

File: README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+## Requirements
+
+[Debian](https://www.debian.org/).
+Maybe this role also works on other apt based systems.
+
+## Role Variables
+
+| Name                   | Required | Description                                                                                                                                          |
+| :--------------------- | :------- | :--------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `cephadm_release_name` | yes      | Release name for `cephadm add-repo --release`, e.g. `quincy`. For possible release names, see: <https://docs.ceph.com/en/quincy/releases/index.html> |
```
Comment (SF2311) [inline README.md]:
```suggestion
| `cephadm_release_name` | yes      | Release name for `cephadm add-repo --release`, e.g. `quincy`. For possible release names, see: <https://docs.ceph.com/en/latest/releases/> |
```

File: meta/main.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+galaxy_info:
+  author: haslersn
+  description: Ansible role to install cephadm; see https://docs.ceph.com/en/quincy/cephadm/
```
Comment (SF2311) [inline meta/main.yml]:
```suggestion
  description: Ansible role to install cephadm; see https://docs.ceph.com/en/latest/cephadm/
```

File: meta/main.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  company: stuvus (https://stuvus.uni-stuttgart.de)
+
+  license: MIT
+
+  min_ansible_version: 2.5
+
+  platforms:
+    - name: Debian
+      versions:
+        - buster
```
Comment (SF2311) [inline meta/main.yml]:
```suggestion
        - bullseye
```

**Review by SF2311 — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+# ansible_cephadm
+
+This role installs [cephadm](https://docs.ceph.com/en/latest/cephadm/) and Docker on Debian.
```
Comment (SF2311) [inline README.md]: There is no task for installing docker

File: meta/main.yml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+  platforms:
+    - name: Debian
+      versions:
+        - bullseye
+
+  galaxy_tags:
+    - ceph
+    - cephadm
+    - docker
```
Comment (SF2311) [inline meta/main.yml]: Not docker related.

File: README.md (hunk 5 above)
Comment (haslersn) [reply README.md]: Right, I removed that because we don't use it.

File: meta/main.yml (hunk 6 above)
Comment (haslersn) [reply meta/main.yml]: Fixed.

**Review by SF2311 — APPROVED** [review]

**Pull request merged**